tests/tests/test_00_multi_tenancy.py

```diff
@@ -27,6 +28,7 @@
 
 
 class TestMultiTenancyEnterprise(MenderTesting):
+    @pytest.mark.enterprise_no_client
```

Comment:
What happens to tests that aren't marked?

Probably they simply won't run ATM, but we can easily add an auto-fixture to make sure it is present. You can check out [this fixture](https://github.com/mendersoftware/meta-mender/blob/bf4be1f27d8929e9fdd706aa29559ef71f6ebe38/tests/acceptance/fixtures.py#L404) for an example of how to do it.

Reply:
@kacf if they are not marked, the tests won't run. It is easy to run those tests using an expression, e.g.:
`-m 'not setup_failover and not running_custom_production_setup  and ...`
If we manage the markers in a python file, we can easily produce such a string to run "non-marked" tests.

Reply:
But would there ever be a test should not be marked? They all will depend on at least one fixture.